bin/ask-autocomplete.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
```

Comment:
my biggest question would be can we save this registry step each time a new command is created? I know this is a subcommand, but is it still possible to load from the main commander object? Based on the current cmd structure, we only need it for ask ,ask smapi, ask utils, ask skill. Is it possible to iterate the command list in bin folder, and dynamically load them?

Reply:
I cannot think of a way. Do you know a way?